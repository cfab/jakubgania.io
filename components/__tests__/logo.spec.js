import { shallowMount } from "@vue/test-utils";
import logo from "../logo";

const factory = () => {
    return shallowMount(logo, {
    });
  };

  describe("Logo", () => {
    test("mounts properly", () => {
      const wrapper = factory();
      expect(wrapper.isVueInstance()).toBeTruthy();
    });
  
    // test("renders properly", () => {
    //   const wrapper = factory();
    //   expect(wrapper.html()).toMatchSnapshot();
    // });
  });