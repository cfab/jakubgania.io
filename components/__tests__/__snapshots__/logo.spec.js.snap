// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders properly 1`] = `
<div class="logo">
  Jakub Gania Software Example Test
</div>
`;
